packages/web/src/pages/artist-coins-launchpad-page/components/LaunchpadBuyModal.tsx

```diff
@@ -404,7 +407,7 @@ export const LaunchpadBuyModal = ({
   // This is because the audio balance hook is polling, so an optimistic change here is not sufficient
   const hasAudioBalanceChanged = useMemo(() => {
     return (
-      prevAudioBalance &&
+      prevAudioBalance !== undefined &&
```

Comment:
this kind of bug is very nasty with numeric inputs, took a while to hunt it down

Reply:
nasty one, good find